tests/src/main/java/ma/glasnost/orika/test/constructor/ConstructorMappingTestCase.java

```diff
@@ -465,8 +470,51 @@ public void testConstructorsWithoutDebugInfo() {
     	Assert.assertEquals(dto1.portX, url.getPort());
     	
     }
-    
-    // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+	@Test
+	public void testFindConstructor_mapParamNames_ok() {
+		ConstructorResolverStrategy constructorStrategy = new SimpleConstructorResolverStrategy() {
+
+			@Override
+			protected String[] mapTargetParamNames(String[] parameterNames) {

```

Comment:
I can also create a new constructor resolver strategy impl in orika-core that handles prefixed constructor param names e.g. PrefixParamConstructorResolverStrategy.